tests/partial_props.rs

```diff
@@ -0,0 +1,54 @@
+use confique::Config;
+use pretty_assertions::assert_eq;
+
+#[test]
+fn partial_props() {
+    #[allow(dead_code)]
+    #[derive(Config)]
+    #[config(partial_attr(derive(derive_more::Debug)))]
+    struct Foo {
+        #[config(default = 1, partial_attr(debug("test {bar:?}")))]
+        bar: u32,
+    }
+
+    use confique_partial_foo::PartialFoo;
```

Comment:
I'd prefer `type PartialFoo = <foo as Config>::Partial` here because I don't want users of this library to rely on the exact module name. Sure, this is a internal test, but still, users might look at this and assume it's ok.

Reply:
Fixed :+1: